core/record/recordExpressionParser/recordEpressionFunctions.js

```diff
@@ -1,8 +1,26 @@
+import * as A from '@core/arena'
+import * as Survey from '@core/survey/survey'
+import * as Category from '@core/survey/category'
+import * as CategoryItem from '@core/survey/categoryItem'
 import * as Record from '@core/record/record'
 import * as Node from '@core/record/node'
 import * as Expression from '@core/expressionParser/expression'
 
-export const recordExpressionFunctions = ({ record }) => ({
+export const recordExpressionFunctions = ({ survey, record }) => ({
+  [Expression.functionNames.categoryItemProp]: (categoryName, itemPropName, ...codePaths) => {
```

Comment:
would it be better to avoid destructuring the codePaths like `...codePaths` ?

Reply:
it makes the usage of the categoryItemProp function easier for the users.
E.g. categoryItemProp('list_name', 'prop_name', 1, 2, 3) instead of something like categoryItemProp('list_name', 'prop_name', Array.of(1,2,3))
(we cannot use the literal notation to initialize arrays now, like [1,2,3], it's not supported by our expression evaluator)